apps/marketplace/components/BuyerSpecialist/BuyerSpecialistResponseFormatsStage.js

```diff
@@ -252,7 +252,7 @@ const BuyerSpecialistResponseFormatsStage = props => (
     />
     <Textfield
       model={`${props.model}.budgetRange`}
-      label="Additional budget information (optional)"
+      label="Additional budget information (optional) Remember to request that sellers provide an offer of discount(s) and to seek better rates of longer-term contracts."
```

Comment:
Let's remove the added content from the header and add it to the description field.
Just so we are keeping it consistent with the [design system's format](https://designsystem.gov.au/components/body/accessibility/)

So add a description field under the Textfield component. 
`description="Remember to request that sellers provide an offer of discount(s) and to seek better rates of longer-term contracts."`

Reply:
Lets increase to text field's input to 50 words rather than 100 characters. Since the 100 character limit would insufficient for buyers. 

Make sure to update the validation field as well.

Reply:
textfield was changed to testarea with validation applied